web/base_viewer.js

```diff
@@ -1358,6 +1358,7 @@ class BaseViewer {
    * @param {PDFPage} pdfPage
    * @param {AnnotationStorage} [annotationStorage] - Storage for annotation
    *   data in forms.
+   * @property {IPDFLinkService} linkService
```

Comment:
This property is only used internally and is not part of the input/output and function signature, so I don't think we should mention it here (see https://github.com/mozilla/pdf.js/pull/14092/commits/bb9c905c5d6d11883888f328d288384e764f19fa#diff-f074a785f6c5ac2ecc4eff615f679c23dde5611034f651f85af67daa254c6fe4R137 where it's also not done).

Reply:
Whoops, this was a case of too much copy-and-pasting; fixed now :-)